packages/@cdktf/provider-generator/lib/get/generator/emitter/attributes-emitter.ts

```diff
@@ -148,6 +151,14 @@ export class AttributesEmitter {
     }
 
     this.code.line(`// Getting the computed value is not yet implemented`);
+    if (type.isSet) {
```

Comment:
What would be required to make it work in non-ts languages?

Reply:
This would require moving away from native array return types in all cases or dynamically generated token types.

When working with jsii, the type of the returned object needs to actually match the type signature of the method.